src/transitions/scroll-transition.js

```diff
@@ -153,7 +153,7 @@ export default class ScrollTransition extends React.Component {
       height: this.props.vertical
         ? listWidth + spacingOffset
         : this.props.slideHeight,
-      width: this.props.vertical ? 'auto' : listWidth + spacingOffset,
```

Comment:
Alot of the perf hit was here for non-vertical, scrolling carousels. It would render slides, then set widths accordingly, which causes layout reflows.